### ISSUE TITLE
Cache policy warning

### ISSUE THREAD

**Issue by erlapso** [GoogleChrome/web.dev#483]
The performance check (PageSpeed test I’m looking at you) is reporting that 23 resources have an inefficient cache policy, but the detail actually shows that the cache policy is over 2 days, sometimes over 30 days

**Comment by robdodson** [comment]
@erlapso can you share a URL we can test to verify?

**Comment by erlapso** [comment]
Sure. https://botsociety.io

**Comment by robdodson** [comment]
Yeah this is also seems like a LH question since it shows up as a failure in DevTools as well. I wonder if the problem is that some of the resources don't have a cache policy, and so it's just showing you all of them.
